### PR TITLE
Control: updated MPC submodule

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/control/submodules/mpc_controller_submodule.cc [hunk 1]
```diff
@@ -48,56 +48,47 @@ bool MPCControllerSubmodule::Init() {
...
 
-  Status status = ProduceControlCommand(&control_command);
+  Status status = ProduceControlCoreCommand(preprocessor_status->local_view(),
+                                            &control_core_command);
   AERROR_IF(!status.ok()) << "Failed to produce control command:"
                           << status.error_message();
-  control_command_writer_->Write(
-      std::make_shared<ControlCommand>(control_command));
+  control_core_writer_->Write(
+      std::make_shared<ControlCommand>(control_core_command));
```
Comment (xiaoxq) [inline modules/control/submodules/mpc_controller_submodule.cc]: `control_core_writer_->Write(control_core_command);` works the same.

Comment (sjiang2018) [reply modules/control/submodules/mpc_controller_submodule.cc]: Fixed. Thank you!

File: modules/control/dag/mpc_module.dag [hunk 2]
```diff
@@ -31,7 +28,6 @@ module_config {
         class_name : "MPCControllerSubmodule"
         config {
             name: "MPCControllerSubmodule"
-            config_file_path: "/apollo/modules/control/conf/calibration_table.pb.txt"
```
Comment (Capri2014) [inline modules/control/dag/mpc_module.dag]: @sjiang2018 Why remove the conf files here?

Comment (sjiang2018) [reply modules/control/dag/mpc_module.dag]: All configs are included in mpc_controller conf file

**Pull request merged**